problems/aram_babken_gagik_khndzorner.py

```diff
@@ -0,0 +1,545 @@
+from manim import *
+from aramanim import *
+
+#diagram config
+BUUF_BETWEEN_PLAYERS = 0.75
+
+#segmen config
+UNIT_SEGMENT_LENGTH = 1.7
+UNIT_SEGMENT_VALUE = 9
+DEFAULT_LABEL_STROKE_WIDTH = 6
+VALUE_TO_SEGMENT_LENGTH = UNIT_SEGMENT_LENGTH / UNIT_SEGMENT_VALUE
+SCALE_DEMERGER = 2
+##
+
+##color
+COLOR_PALETTE = [
+    '#88F7E2',
+    '#44D492',
+    '#F5EB67',
+    '#FFA15C',
+    '#FA233E'
+]
+##
+
+##names
+NAME_1 = r'Արամ'
+NAME_2 = r'Բաբկեն'
+NAME_3 = r'Գագիկ'
+##
+
+##texts
+TEXT_2 = [r'― Արամի քառապատիկից ', r'6', r'-ով ավելի']
+TEXT_3 = [r'― Բաբկենի կեսից ', r'1', r'-ով պակաս']
+
+##animation config
+RUN_TIME = 2
+WAIT_TIME = 1.5
+MEDIUM_LAG_RATIO = 0.55
+SMALL_LAG_RATIO = 0.25
+BIG_LAG_RATIO = 0.75
+
+##main functions
+
+def unit_segment():
+    label_text = r"{:,}".format(int(UNIT_SEGMENT_VALUE))
+    s = Segment(
+        ORIGIN, UNIT_SEGMENT_LENGTH*RIGHT,
+        label_text, DEFAULT_LABEL_FONT_SIZE,
+        stroke_width=DEFAULT_LABEL_STROKE_WIDTH, color=COLOR_PALETTE[1]
+    )
+    s.label.add_updater(lambda mob: s.update_label_pos())
+    return s
+
+def segment_from_value(value):
+    label_text = r"{:,}".format(int(value))
+    s = Segment(
+        ORIGIN, VALUE_TO_SEGMENT_LENGTH*SCALE_DEMERGER*value*RIGHT,
+        label_text, DEFAULT_LABEL_FONT_SIZE,
+        stroke_width=DEFAULT_LABEL_STROKE_WIDTH, color=COLOR_PALETTE[4]
+    )
+    s.label.add_updater(lambda mob: s.update_label_pos())
+    return s
+
+def unit_segment_with_additional(additional):
+    label_text = r"{:,}".format(int(additional))
+    s = Segment(
+        ORIGIN, VALUE_TO_SEGMENT_LENGTH*SCALE_DEMERGER*additional*RIGHT,
+        label_text, DEFAULT_LABEL_FONT_SIZE,
+        stroke_width=DEFAULT_LABEL_STROKE_WIDTH, color=COLOR_PALETTE[4]
+    )
+    s.label.add_updater(lambda mob: s.update_label_pos())
+    return [unit_segment(), s]
+
+def from_player_segment(player):
+    s = Segment(
+        player.get_edge_center(LEFT),
+        player.get_edge_center(RIGHT),
+        stroke_width=DEFAULT_LABEL_STROKE_WIDTH
+    )
+    return s
```

Comment:
Can we add those to `Segment ` class as classmethods, in that case those will be useful for others too. e.g.

```python
class Segment:
    # some code goes here
   
    @classmethod
    def unit_segment(cls, value, length, font_size, stroke_width, color):
        label_text = r"{:,}".format(int(value))
        s = cls(
            ORIGIN,
            length * RIGHT,
            label_text,
            font_size,
            stroke_width=stroke_width,
            color=color
        )
        s.label.add_updater(lambda mob: s.update_label_pos())
        return s
```

Here `cls` refers to `Segment` class.
You'll call the function as `Segment.unit_segment(9, 1.7, other arguments, ...)`

If it won't be used by others, or will be too much work, then don't do it yet.